generators/lib/distros/octane.js

```diff
@@ -31,21 +32,29 @@ function init() {
     done();
   };
 
-  module.modifyComposer = function(yo, options, localComposer, isNewProject, done) {
+  module.loadComposer = function(yo, options) {
+    var composer = {};
+    var file = yo.templatePath('drupal/drupal/' + options.drupalDistroRelease + '/composer.json');
```

Comment:
This code is very similar to the code in drupal.js.  I'd recommend another helper utility function to handle the common templatePath and file reading.